contracts/pcv/IERC20HoldingPCVDeposit.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/// @title ERC20HoldingPCVDeposit
+/// @notice PCVDeposit that is used to hold ERC20 tokens as a safe harbour. Deposit is a no-op
+interface IERC20HoldingPCVDeposit {
+    /// @notice Token which the balance is reported in
+    function token() external view returns (IERC20);
+
+    ///////   READ-ONLY Methods /////////////
```

Comment:
can get rid of this

Reply:
what's the reason we should remove this?